types/aws-lambda/index.d.ts

```diff
@@ -465,17 +466,25 @@ export type CloudFormationCustomResourceResponse =
     | CloudFormationCustomResourceFailedResponse;
 
 /**
- * See https://docs.aws.amazon.com/lambda/latest/dg/eventsources.html#eventsources-scheduled-event
+ * See https://docs.aws.amazon.com/eventbridge/latest/userguide/aws-events.html
  */
-export interface ScheduledEvent {
+export interface EventBridgeEvent<T> {
+    "detail-type": string;
     account: string;
+    detail: T;
+    id: string;
     region: string;
-    detail: any;
-    'detail-type': string;
+    resources: string[];
     source: string;
     time: string;
-    id: string;
-    resources: string[];
+    version: string;
+}
+
+/**
+ * See https://docs.aws.amazon.com/eventbridge/latest/userguide/event-types.html#schedule-event-type
+ */
+export interface ScheduledEvent extends EventBridgeEvent<{}> {
+    'detail-type': "Scheduled Event";
```

Comment:
Instead of `{}`, use `any` here to preserve the previous type of `detail`: otherwise you are breaking handlers that just do `event.detail.someValue`...

Also, not sure this is an improvement or not ,but an alternative I've used is something like:

```ts
export interface EventBridgeEvent<Type, Detail> {
  "detail-type": Type;
  detail: Detail;
  ...
}
export type ScheduledEvent = EventBridgeEvent<"Scheduled Event", any>;
```

This makes it easier to declare large sets of these event types.